zh/enable-tidb-cluster-auto-scaling.md

```diff
@@ -1,41 +1,41 @@
 ---
-title: 启用 TidbCluster 弹性伸缩
-summary: 介绍如何使用 TidbCluster 的弹性伸缩能力。
+title: 对 Kubernetes 上的 TiDB 集群启用自动弹性伸缩
+summary: 介绍如何对 Kubernetes 上的 TiDB 集群启用自动弹性伸缩。
 aliases: ['/docs-cn/tidb-in-kubernetes/dev/enable-tidb-cluster-auto-scaling/']
 ---
 
-# 启用 TidbCluster 弹性伸缩
+# 对 Kubernetes 上的 TiDB 集群启用自动弹性伸缩
 
-在 Kubernetes 平台上，有原生的弹性伸缩 API: [Horizontal Pod Autoscaler](https://kubernetes.io/zh/docs/tasks/run-application/horizontal-pod-autoscale/)。基于 Kubernetes，TiDB 5.0 起支持了全新的弹性调度算法。与之相应的，在 TiDB Operator 1.2 及以上版本中，TiDB 集群可以凭借 Kubernetes 平台本身的特性来开启弹性调度的能力。本篇文章将会介绍如何开启并使用 TidbCluster 的弹性伸缩能力。
-
-## 开启弹性伸缩特性
+基于 Kubernetes，TiDB 自 v5.0 起支持了全新的弹性伸缩算法。相应地，在 TiDB Operator 1.2 及以上版本中，TiDB 集群可以凭借 Kubernetes 平台本身的特性支持开启自动弹性伸缩。本文介绍如何对 Kubernetes 上的 TiDB 集群开启和使用自动弹性伸缩。
```

Comment:
```suggestion
基于 Kubernetes，TiDB 自 v5.0 起支持了全新的弹性伸缩算法，可以凭借 Kubernetes 平台本身的特性，开启自动弹性伸缩。本文介绍如何对 Kubernetes 上的 TiDB 集群开启和使用自动弹性伸缩。
```
This pull request only applies to v1.2 and later versions, so the version description might not be necessary.